packages/core/platform/bundles/paas-full.yaml

```diff
@@ -399,3 +399,11 @@ releases:
   releaseName: reloader
   chart: cozy-reloader
   namespace: cozy-reloader
+
+- name: velero
+  releaseName: velero
+  chart: cozy-velero
+  namespace: cozy-velero
+  privileged: true
+  optional: true
+  dependsOn: [monitoring-agents]
```

Comment:
should depends also on `cilium` and `kube-ovn`

Please also add velero as optional component to `distro-full` and `distro-hosted`